src/_dialog.scss

```diff
@@ -0,0 +1,78 @@
+.dialog {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	width: 800px;
+	height: 500px;
+	align-items: center;
+	border: 1px solid #ffe81f;
+	border-radius: 10px;
+	gap: 10px;
+
+	&__title {
+		margin-top: 1em;
+		font-size: 4.8rem;
+		color: #ffe81f;
+		text-transform: uppercase;
+		font-family: serif;
+		text-align: center;
+	}
+
+	&__description {
+		text-transform: uppercase;
+		padding-inline: 2.5em;
+		line-height: 2.9rem;
+		font-size: 1.8rem;
+		color: aliceblue;
+		text-align: center;
+	}
+}
+
+.form {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	align-self: center;
+	margin-top: .5em;
+	width: 70%;
```

Comment:
A co z wersją mobilną? Będziemy robić? Jak myślisz, przyda się?

Reply:
Wersji mobilnej nie robiłem w tym przypadku bo wszystko ładnie schodzi aż do 390px. Chyba, że powinno się schodzić do jeszcze niższych rozdzielczości.
![Responsive Viewer – Brave 21 07 2024 13_25_12](https://github.com/user-attachments/assets/19c4c3bc-a1c1-40b0-a33c-ca2e8f2934c9)

Reply:
Wersji mobilnej w tym przypadku nie dodawałem, bo wszystko ładnie schodzi do 380px. Chyba, że powinno się jeszcze niżej. Jeśli tak to bez problemu zrobię :)
![Responsive Viewer – Brave 21 07 2024 13_25_12](https://github.com/user-attachments/assets/8a9abd40-6d27-4560-bd36-a54b68966dbb)